content.yml

```diff
@@ -92,15 +92,21 @@ ecosystem:
     technological and legal innovation etc.
   subtitle: "Go to:"
   linka1: "Neufunds Ecosystem Impact Page"
+  linka1ref: "#"
```

Comment:
It can be done, but i think in the end we won't use loops to generate any of this content. 
However will do that now